trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/compressibility_drag_wing.py

```diff
@@ -68,12 +66,6 @@ def compressibility_drag_wing(state,settings,geometry):
     # unpack wing
     t_c_w   = wing.thickness_to_chord
     sweep_w = wing.sweeps.quarter_chord
-    
-    # Currently uses vortex lattice model on all wings
-    if wing.tag=='main_wing':
-        cl_w = wing_lifts
-    else:
-        cl_w = 0
         
     cos_sweep = np.cos(sweep_w)
 
```

Comment:
There's a commented out piece of code below ...

Reply:
I've restored it to produce what I believe was the desired functionality.